mne/preprocessing/ica.py

```diff
@@ -1813,14 +1814,15 @@ def find_bads_muscle(self, inst, threshold=0.5, start=None,
 
         # typical muscle slope is ~0.15, non-muscle components negative
         # so logistic with shift -0.5 and slope 0.25 so -0.5 -> 0.5 and 0->1
+        slope_score = expit((slopes + 0.5) / 0.25)
         # focus distance is ~65% of max electrode distance with 10% slope
         # (assumes typical head size)
+        focus_score = expit((focus_dists - 0.65) / 0.1)
         # smoothnessness is around 150 for muscle and 450 otherwise
         # so use reversed logistic centered at 300 with 100 slope
+        smoothness_score = 1 - expit((smoothnesses - 300) / 100)
         # multiply so that all three components must be present
-        scores = (1 / (1 + np.exp(-(slopes + 0.5) / 0.25))) * \
-            (1 / (1 + np.exp(-(focus_dists - 0.65) / 0.1))) * \
-            (1 - (1 / (1 + np.exp(-(smoothnesses - 300) / 100))))
+        scores = slope_score * focus_score * smoothness_score
```

Comment:
@alexrockhill can you double-check this change?

Reply:
this looks correct to me

Reply:
Looks good, again sorry for the late reply